jekyll/_cci2/deploy-to-amazon-sagemaker.adoc

```diff
@@ -0,0 +1,364 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Deploy to Amazon SageMaker
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A short page description goes here
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available. The link:https://circleci.com/developer/orbs/orb/circleci/aws-sagemaker[SageMaker orb] is also in **open preview**.
+
+This how-to guide walks you through the steps to set up and use the Amazon SageMaker orb to deploy and release models to SageMaker. You will track releases in the CircleCI release dashboard. This gives you the ability to manage promotions and rollbacks. You will orchestrate model deployments to endpoints across different environments. The link:https://github.com/CircleCI-Public/sagemaker-deploy-examples#[example project repository] allows you to train a new model package version and then deploy it across multiple environments.
+
+For further information about the orb and the releases feature see the following:
+
+* xref:/release/releases-overview#[Releases overview]
+* link:https://circleci.com/developer/orbs/orb/circleci/aws-sagemaker[Amazon SageMaker orb]
```

Comment:
Would we ever link to a blog article from docs?

Reply:
absolutely yes

Reply:
Ok I've got the URL but it's not live yet. It should go live roughly at the same time as these docs so I think it's ok to include! 
https://circleci.com/blog/amazon-sagemaker-deployment-orchestration

Reply:
added